src/ws/cockpitsshtransport.c

```diff
@@ -330,6 +330,10 @@ cockpit_ssh_authenticate (CockpitSshData *data)
               g_message ("%s: password auth worked, but server wants more authentication",
                          data->logname);
               break;
+            case SSH_AUTH_AGAIN:
+              g_message ("%s: server asked you to try logging in again",
```

Comment:
How about changing the message to say "password auth failed: server asked for retry"

Because at this point we can't reprompt the user, it's really just a failure.